File: Java/Tagi HTML.java
import java.util.Scanner;

class Main
{
    public static void main(String[] args)
    {
        Scanner input = new Scanner(System.in);
        while (input.hasNextLine()) 
        {
            boolean bigLetters = false;
            String output = "";
            String line = input.nextLine();
            char[] words = line.toCharArray();
            for (int i = 0; i < words.length; i++) 
            {
                char word = words[i];
                if (word == '<') 
                {
                    bigLetters = true;
                    output += word;
                } 
                else if (word == '>') 
                {
                    bigLetters = false;
                    output += word;
                } 
                else if (bigLetters)
                    output += Character.toUpperCase(word);
                else
                    output += word;
            }
            System.out.println(output);
        }
        input.close();
    }
}

File: Python/Tagi HTML.py
while True:
    try:
        big_letters = False
        output = ''
        line = input() 
        for symbol in line:
            if symbol == '<': 
                big_letters = True
                output += symbol
            elif symbol == '>':
                big_letters = False 
                output += symbol
            elif big_letters:
                output += symbol.upper()
            else:
                output += symbol 
        print(output)
    except:
        break
